### PR TITLE
Fix PED bugs

### PR DESCRIPTION
**Pull request by Slipperybara**
Fix PED bugs in UG and DG
Update some javadoc comments

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -1482,12 +1487,14 @@ maintainability of our code. We went the extra mile for passion and our own lear
 ---------------------------------------------------------------------------------------------------------------------
 ## **Appendix: Planned Enhancements**
```
Comment (jinhanjinhan) [inline docs/DeveloperGuide.md:1489]: Screenshot 2024-11-11 at 8 51 37 PM
I think for this part, we need to add in the team size as well

File: docs/DeveloperGuide.md [hunk 2]
```diff
@@ -1496,16 +1503,22 @@ a “Timing” attribute to tutorials, enabling administrators to view and manag
...
+student into one tutorial at a time. We plan to add functionality for batch enrollments and unenrollments, allowing
+administrators to enroll or unenroll multiple students from a tutorial at once. As well as enroll and unenroll
+a student from multiple tutorials.
+8. **Enhance Error Message specificity**: Currently, certain error messages related to invalid user input are either too
+vague or not displayed at all. Specifically, if a user includes a prefix that does not fall within the command's scope,
+the command will not return an error message. Instead, it might process the prefix as a string input and passes it to
+other fields, potentially causing additional errors. Moving forward, we plan to restrict the use of invalid prefixes based on
+the specific command to ensure more precise error handling.
+9. **Allowing Multiple Attendance Records**: Currently, attendance can only be recorded once per tutorial per week.
+This is to ensure user do not accidentally key in the wrong attendance. We plan to allow multiple attendance within the
```
Comment (gladwintan) [inline docs/DeveloperGuide.md:1519]: can add "which results in duplicate weekly attendance"?

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (Slipperybara) [reply docs/DeveloperGuide.md:1489]: ok good catch

File: docs/DeveloperGuide.md [hunk 3]
```diff
@@ -1496,16 +1503,22 @@ a “Timing” attribute to tutorials, enabling administrators to view and manag
...
-multiple students from a tutorial at once.
-8. **Batch Enroll/Unenroll for Multiple Tutorials**: Currently, students can only be enrolled or unenrolled from one
-tutorial at a time. We plan to add functionality to support enrolling or unenrolling a student from multiple tutorials
-simultaneously, improving efficiency.
-9. **Prevent Duplicate Attendance Records**: Currently, attendance can be added multiple times even if it’s within the
-same week for the same tutorial participation. We plan to add a check to prevent duplicate attendance within the same week, ensuring accurate attendance tracking.
+6. **Support For Special Characters in Names**: Currently, special characters such as '/' are not allowed in the name
+field.
+In the future, we plan to enhance the application to allow such characters in name inputs.
+7. **Batch Enroll/Unenroll for Multiple Students and Tutorials**: The current system allows enrolling or unenrolling one
```
[the remark this answers is not in the document]
Comment (Slipperybara) [reply docs/DeveloperGuide.md:1509]: ok I will remove batch add tutorial, think batch add student is more useful

File: docs/DeveloperGuide.md (hunk 2 above)
Comment (Slipperybara) [reply docs/DeveloperGuide.md:1519]: ok will add to this line 'This is to ensure user do not accidentally key in the wrong attendance, which results in duplicate weekly attendance'

**Review by gladwintan — APPROVED** [review]
LGTM

**Review by jinhanjinhan — APPROVED** [review]
Looks good

**Review by amitha2210 — APPROVED** [review]
LGTM

**Pull request merged**